### ISSUE TITLE
Add parameter to set validation_interval for logging

### ISSUE THREAD

**Issue by eileen-kuehn** [cirKITers/masKIT#59]
In the current logging process validation results are logged after the training. However, it might be of interest to see how the loss and accuracy develops over number of training steps. We might therefore consider adding an additional parameter `validation_interval` or even a fixed factor for the current `log_interval`.

**Comment by eileen-kuehn** [comment]
This has been solved in #62 already.
